.github/workflows/java-pr.yml

```diff
@@ -79,7 +79,7 @@ jobs:
     name: Dataflow Templates Integration Tests
     needs: [spotless_check, java_build, java_unit_tests]
     timeout-minutes: 180
-    runs-on: self-hosted
+    runs-on: [self-hosted, it]
```

Comment:
nit: Maybe adding a comment that runner has to have both can be useful here.

I [had to lookup the behavior](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on)

> If you specify an array of strings, your workflow will execute on any runner that matches all of the specified runs-on values.

Reply:
Done